client/src/modules/lead/EditLead/EditLeadHeader/AdditionalActionsPopover/AdditionalActionsPopover.tsx

```diff
@@ -0,0 +1,49 @@
+import * as React from 'react';
+import { Popover } from 'reactstrap';
+import * as styles from './AdditionalActionsPopover.css';
+
+interface Props {
+  target: string;
+  isOpen: boolean;
+  leadId: string;
+
+  toggle(): void;
+
+  deleteLead(leadId: string): void;
+}
+
+class AdditionalActionsPopover extends React.Component<Props> {
+
+  public render() {
+    return (
+      <Popover
+        className={styles.popover}
+        placement="bottom-end"
+        target={this.props.target ? this.props.target : ''}
+        isOpen={this.props.isOpen}
+        toggle={this.props.toggle}
+      >
+        <div className={styles.popoverBody}>
+          <div className={styles.actions}>
+            <div
```

Comment:
reformat code according to convention(you dont have a lot of props in component use declaration on simple line)